app/controllers/sequences_controller.rb

```diff
@@ -144,12 +157,11 @@ def destroy
     else
       flash_warning(:permission_denied.t)
     end
+
     if @back == "index"
-      # redirect_with_query(
-      #   action: :index_sequence
-      # )
-      redirect_to sequences_index_sequence_path(@sequence.id,
-                                                q: get_query_param)
+      redirect_with_query(action: :index_sequence)
```

Comment:
I restored the `redirect` to what was in master because I could not get the helper path to work; it returns a period where we need a slash.
```ruby
$ sequences_index_sequence_path(@sequence.observation.id)
> "/sequences/index_sequence.651436721"
```
where what we want is
```ruby
> "/sequences/index_sequence/651436721"
```

Reply:
Joe - Try 
`sequences_index_sequence_path(id: @sequence.observation.id)` 

What i'm seeing in rake routes is:

```
                           sequences_observation_index GET      /sequences/observation_index(.:format)                                                   sequences#observation_index
                                                       GET|POST /sequences(/observation_index(/:id))(.:format)                                           sequences#observation_index {:id=>/\d+/}
```

If you like i'll try switching this over to inheriting from IntegrationControllerTest and try it. I wouldn't be surprised if it magically works with the newer code, without explicitly stating that it's an `id:` param. 

In any case I think it's wise to get a workflow going where we are switching all these controller tests to the current dependency.

Reply:
Thanks for showing me the right syntax.
Even better, it seems that if @sequence is defined, one need not specify `id` in the arguments.
Fixed in Commit #778de18